### PR TITLE
DTM 1019.1.0 user guide

### PR DESCRIPTION
**Pull request by spau6**
Added Localization related changes.
Added File sanitization related information.

### CODE REVIEW

File: content/dtm/localization.md [hunk 1]
```diff
@@ -14,10 +14,10 @@ helpcontent:
   Use the **Localization** feature to add translations for asset properties or asset models, as well as to add custom translations for the existing static text in the DTM application.
 
 
-   Once you created all asset properties and asset models, their names are added as keys in the **Localization** page. You can add further translations as required.  
+   Once you created all asset properties and asset models, their names are added as Identifier in the **Localization** page. You can add further translations as required.
```
Comment (MWindel) [inline content/dtm/localization.md:17]:
```suggestion
   Once you created all asset properties and asset models, their names are added under **Identifier** in the **Localization** page. You can add further translations as required.  
```

File: content/dtm/localization.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ If you want to add translations for the created [asset properties](/dtm/asset-ty
 
 ![Main view localization page](/images/dtm/localization/dtm-localization-main-page.png)
 
-Once you create an asset property or an asset model, their names are added automatically as keys in the **Localization** page. You can further add the translations as required.
+Once you create an asset property or an asset model, their names are added automatically as Identifier in the **Localization** page. You can further add the translations as required.
```
Comment (MWindel) [inline content/dtm/localization.md:62]:
```suggestion
Once you create an asset property or an asset model, their names are added automatically under **Identifier** in the **Localization** page. You can further add the translations as required.
```

File: content/dtm/localization.md [hunk 3]
```diff
@@ -71,11 +71,11 @@ To add a new translation in the DTM application, you can add the text as key and
 
 Click **Apply** to apply the changes.
 
-### To add new keys for translations {#to-add-new-keys-for-translations}
+### To add new Identifier for translations {#to-add-new-keys-for-translations}
```
Comment (MWindel) [inline content/dtm/localization.md:74]:
```suggestion
### To add new identifier for translations {#to-add-new-keys-for-translations}
```

File: content/dtm/localization.md [hunk 4]
```diff
@@ -71,11 +71,11 @@ To add a new translation in the DTM application, you can add the text as key and
...
 
-Add new keys and their respective translations using the **Add term** option on the top right.
+Add new Identifier and their respective translations using the **Add translation** option on the top right.
```
Comment (MWindel) [inline content/dtm/localization.md:76]:
```suggestion
Add a new identifier and their respective translations using the **Add translation** option on the top right.
```

File: content/dtm/localization.md [hunk 5]
```diff
@@ -84,7 +84,7 @@ Next, click **Apply** to apply the changes.
 
 ### To add and edit translations {#to-add-and-edit-translations}
 
- You can add or edit translations for all the keys in two ways:
+ You can add or edit translations for all the Identifier in two ways:
```
Comment (MWindel) [inline content/dtm/localization.md:87]:
```suggestion
 You can add or edit translations for every identifier in two ways:
```

File: content/dtm/localization.md [hunk 6]
```diff
@@ -93,13 +93,13 @@ Next, click **Apply** to apply the changes.
 
 or:
 
-1. Click the edit icon in each row to open the **Edit term translation** dialog.
+1. Click the actions button edit icon in each row to open the **Edit translations** dialog.
```
Comment (MWindel) [inline content/dtm/localization.md:96]:
```suggestion
1. Click the edit icon in each row to open the **Edit translations** dialog window.
```

File: content/dtm/localization.md [hunk 7]
```diff
@@ -93,13 +93,13 @@ Next, click **Apply** to apply the changes.
...
 2. Add or edit the translation for any language.
 3. Click **Save**.
 
 Click **Apply** to apply the changes.
 
-Once all your translations are done, you can change the language under **User settings** to see the translations in action. For more information see [To change user settings](/get-familiar-with-the-ui/user-settings/#to-change-user-settings).
+Once all your translations are done, you can change the language under **UI settings** to see the translations in action. For more information see [To change user settings](/get-familiar-with-the-ui/user-settings/#to-change-user-settings).
```
Comment (MWindel) [inline content/dtm/localization.md:102]: @spau6 Can you provide a sample from the new UI which shows the new settings name?

Comment (spau6) [reply content/dtm/localization.md:102]: image

**Comment by MWindel** [comment]
@spau6 Please get a team approval first, then I can approve for merge.

**Review by lra-sag — APPROVED** [review]

**Review by MWindel — APPROVED** [review]

**Pull request merged**